src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,39 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  name = '';
+  health = 100;
```

Comment:
- You don't need to set name here, just set it inside a constructor
- Avoid magic values such as 100, etc. Create constants